shimmingtoolbox/masking/threshold.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Image thresholding API
+"""
+
+import glob
+import os
+import nibabel as nib
+import pathlib
+import tempfile
+import logging
+
+from shimmingtoolbox.utils import run_subprocess
+
+
+def threshold(data, thr=30):
```

Comment:
Is it necessary to have a function that only does this? It adds another layer of complexity and linking for the executer. If it will be used at multiple places or do other things then it is alright, but this function seems quite short.

Reply:
probably not, but at some point we might have "smarter" thresholding methods à la Otsu, patch-based thresholding, etc. so the idea is just to get started with something trivial so we can move forward with our example scenario

Reply:
Good, just wanted to be sure.